src/Prima/Pyxis/Tooltip.elm

```diff
@@ -58,32 +59,39 @@ type TooltipType
     | Right
 
 
+type TooltipKind
```

Comment:
Di solito (es. https://github.com/primait/pyxis-components/blob/master/src/Prima/Pyxis/Accordion.elm)
abbiamo usato il kind come metodo di configurazione. Qui diciamo che non mi convince troppo avere un `withBrand`, `withError` ecc. 

Forse si potrebbe fare in modo che i metodi `top, left, right, bottom` spariscano in favore di `withPositionTop, withPositionLeft ecc.`. A questo punto potremmo poi creare 3 funzioni di configurazione: `brand, error, warning`, da usare così: 
```
myTooltipBrand = 
    Tooltip.brand 
        |> Tooltip.withPositionTop
        ...

myTooltipError = 
    Tooltip.error 
        |> Tooltip.withPositionLeft
        ...
```

Che ne pensi @damjack ?

Reply:
In effetti mi piace di più come proponi. Non so se è troppo rottura e magari lasciamo anche le precedenti specifiche ma indicando che sono deprecate e che saranno rimosse.